prov/efa/src/efa_cq.h

```diff
@@ -2,6 +2,8 @@
 /* SPDX-FileCopyrightText: Copyright Amazon.com, Inc. or its affiliates. All rights reserved. */
 
 #include "efa.h"
+#include "efa_av.h"
+#include "efa_cntr.h"
```

Comment:
why do you need cntr in this header file? If you need cntr in efa_cq.c, you should include it explicitly in it.